### ISSUE TITLE
New option: Other playername text + color

### ISSUE THREAD

**Issue by nco2k** [mrbuilder1961/WheresMyChatHistory#161]
it would be cool if you could introduce two more options for the Chat Interface:
- Other playername text
- Other playername color

**Comment by mrbuilder1961** [comment]
it sounds like your version isn't working..?

in the latest releases you should already be able to change these if the server hasn't already configured them. they're right next to each other under chat interface > player name text/color, and they apply to all players.

**Comment by nco2k** [comment]
> and they apply to all players.

yes, and thats exactly my point. i would like to visually differentiate between **other** players messages and my own. :D
